sdk/examples/client/node_api_core/10_get_output_raw.rs

```diff
@@ -14,26 +19,25 @@ use iota_sdk::{
 
 #[tokio::main]
 async fn main() -> Result<()> {
+    // If not provided we use the default node from the `.env` file.
+    dotenvy::dotenv().ok();
+
     // Take the node URL from command line argument or use one from env as default.
-    let node_url = std::env::args().nth(1).unwrap_or_else(|| {
-        // This example uses secrets in environment variables for simplicity which should not be done in production.
-        dotenvy::dotenv().ok();
-        std::env::var("NODE_URL").unwrap()
-    });
+    let node_url = env::args().nth(2).unwrap_or_else(|| env::var("NODE_URL").unwrap());
 
-    // Create a client with that node.
+    // Create a client.
     let client = Client::builder().with_node(&node_url)?.finish().await?;
 
-    // Take the output ID from command line argument or use a default one.
-    let output_id =
-        OutputId::from_str(&std::env::args().nth(2).unwrap_or_else(|| {
-            String::from("0xb66fd384cb5755668f1890ea2e41d699db9cf32f3bc422ad3c24ffeb9c7f01d00000")
-        }))?;
+    // Take the output id from the command line, or panic.
+    let output_id = env::args()
+        .nth(1)
+        .expect("missing example argument: OUTPUT ID")
```

Comment:
Should we use a "random" output id from the indexer if nothing was provided? (If so in another PR)

Reply:
yeah, but it would mix core api with indexer api examples, or at least it would blur the clear separation a bit IMO.

Reply:
Ah yeah good point, guess it's fine then